### ISSUE TITLE
Attachments not scrollable when overflowing the container

### ISSUE THREAD

**Issue by thescottyq** [BookStackApp/BookStack#2887]
We have a couple of pages that have a long list of attachments and when you view the attachments list in edit mode it prevents the whole page from scrolling. This effects Chromium, FireFox and Opera on a Windows PC, however works intermittently on Safari on both Mac and iPadOS 

You can fix this by manually editing the raw HTML within the browsers “inspector” mode and adding the following style to the DIV which is **component=“sortable list”**

_style="overflow-y:scroll;height:100vh"_  

A simple fix for this would seem to be to add this CSS to the customer header option within the GUI however we cannot seem to find a way to select that component in CSS. 

Has anyone else encountered this and know of a suitable permanent fix please? 

![C75E6351-DDD7-49A4-A364-48E095690F11](https://user-images.githubusercontent.com/13838125/129692000-b7d6517d-6cb5-4db9-a8e9-39709d7c3d8e.jpeg)

**Comment by JamesTimmins-ops** [comment]
this can be edited at /var/www/bookstack/resources/views/attachments/manager-list.blade.php open a text editor for it (nano or vi) then add style="overflow-y:scroll;height:850vh;" to the component="sortable-list" 

the div will look like this:

<div component="sortable-list" option:sortable-list:handle-selector=".handle" style="overflow-y:scroll;height:850vh;">

**Comment by thescottyq** [comment]
Cheers @JamesTimmins-ops 

Could this be pushed out to live repo please guys?

**Comment by ssddanbrown** [comment]
Note: You can use the [visual theme system](https://github.com/BookStackApp/BookStack/blob/master/dev/docs/visual-theme-system.md) to implement the @JamesTimmins-ops solution without editing core files.

Alternatively, Something like below should work in custom HTML head:

```html
<style>
.floating-toolbox .files div[component="sortable-list"] {
   overflow-y:scroll
   height:100vh;
}
</style>
```

Otherwise, Have assigned to the next feature release.

Note: The implementation solution will ideally need to be flexible instead of using fixed `#vh` units.

**Comment by ssddanbrown** [comment]
Hi @thescottyq,

Just coming to look at this one, but I have not been able to find an issue. Have tested on Firefox and Chrome on Fedora, In addition to Safari on iPad. In each, the I'm able to scroll the attachments sidebar while still being able to scroll page content as expected.

In regards to your comment:

> when you view the attachments list in edit mode it prevents the whole page from scrolling

What do you mean by page? As in the page content within the editor? If so, which editor are you using? This view has a full-height flexible layout where the webpage itself does not usually scroll.

**Comment by thescottyq** [comment]
Hi Dan, thanks for looking at this further. To confirm your suggestion of adding the following to our customer header resolved the issue.

_<style>
.floating-toolbox .files div[component="sortable-list"] {
   overflow-y:scroll
   height:100vh;
}
</style>_

The issue only seems to be on Windows Machines and is specifically when you are editing a page. If you have a long list of attachments the list just does not scroll 

<img width="1685" alt="Pasted_Image_24_08_2021__14_30" src="https://user-images.githubusercontent.com/13838125/130626171-5c94bfa4-ab3e-40c9-8db3-30fe70ed7f4a.png">

**Comment by ssddanbrown** [comment]
Tested this again across Firefox and Chrome/Edge across Fedora Linux and Windows 10, cannot reproduce, so will therefore close this off as I can only assume something else is at play.
Happy to address if a specific issue can be found but I'd need to be able to reproduce the scenario as to not be blindly applying fixes.

**Comment by PvandenBerg77** [comment]
I have the same problem. 

Version:
- Bookstack v22.11

Browsers:
- Tested on Edge 107.0.1418.62 and Chrome 108.0.5359.72

Operating systems:
- Windows Server 2022 Datacenter 21H2
- Windows 10 Pro 21H2

Steps taken:
Open the Book from the Shelve and then navigate to the page holding the attachments. There Click on edit. Then Click on any item to open de side bar where you can enter tags, add attachments etc.

**Comment by ssddanbrown** [comment]
Reopening. @PvandenBerg77 Can you confirm your operating system type/version used for these tests?

**Comment by ssddanbrown** [comment]
Can now confirm reproduction on Firefox on Ubuntu and Chrome on Windows 10. Quite strange, maybe something was rolling out, had further recent confirmations in  #3913. Have assigned to be addressed as part of next patch release.

**Comment by ssddanbrown** [comment]
Alrighty, found I had broken overflow due to changes in v22.11. 
Patched in a521f418382aaec18ed6fce997a2b204a9a7eb56, will be part of the next patch release.

**Comment by CrackedBatch** [comment]
@ssddanbrown Thank you for the quick fix!
